hooks/playbooks/ironic_inject_tls.yml

```diff
@@ -0,0 +1,117 @@
+---
+- name: Inject CA bundle in ironic-python-agent and create images
+  hosts: "{{ cifmw_target_hook_host | default('localhost') }}"
+  gather_facts: false
+  vars:
+    _namespace: openstack
```

Comment:
this one is provided by the run_hook/playbook.yml:
https://github.com/openstack-k8s-operators/ci-framework/blob/43072d16e25c0ad62993e027ed20eb5b0b5024ea/roles/run_hook/tasks/playbook.yml#L34-L35

you want `namespace`.

Reply:
Done.